src/Indicator.jsx

```diff
@@ -13,24 +18,100 @@ class Indicator extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeArea: {},
+      selectedOption: null,
+      biomesByBlockData: [],
+      connError: false,
     };
   }
 
   componentDidMount() {
     const { activeArea } = this.props;
-    this.setState({ activeArea });
+    const areaId = (activeArea && activeArea.name) ? activeArea.name : null;
+    if (areaId) {
+      this.loadBiomes(areaId);
+    }
+  }
+
+  /**
+   * Load biomes for selected area from RestAPI
+   *
+   * @param {string} areaId id for selected area
+   */
+  loadBiomes = (areaId) => {
+    RestAPI.requestBiomesByArea(areaId)
```

Comment:
Just a notice, this endpoint will probably disappear.

Reply:
Ok, so we should define how to populate the biomes list in that case.